getting-started.md

```diff
@@ -1,15 +1,16 @@
 # Getting Started
 
-A TimescaleDB behaves in many respects like a standard PostgreSQL database. It:
-- Coexists with other TimescaleDBs and PostgreSQL databases on a PostgreSQL
-server.
+A TimescaleDB behaves in many respects like a standard PostgreSQL
+database. It:
+
+- Coexists with other TimescaleDBs and PostgreSQL databases on a PostgreSQL server.
 - Uses SQL as its interface language.
 - Contains standard database objects like tables, indexes, and triggers.
 - Uses common PostgreSQL connectors to third-party tools.
 
-The way the database accomplishes this synchronicity is through its packaging
-as a PostgreSQL extension, whereby a standard PostgreSQL database is
-transformed into a TimescaleDB.
+The way the database accomplishes this synchronicity is through its
+packaging as a PostgreSQL extension, whereby a standard PostgreSQL
+database is transformed into a TimescaleDB.
```

Comment:
```suggestion
database is transformed into a TimescaleDB database.
```

FWIW, I find this sentence odd and superfluous so could just as well remove altogether, imo.